themes/default/content/docs/intro/vs/aws-cdk/_index.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Pulumi vs. AWS CDK
+meta_desc: Pulumi and AWS CDK are alike in some ways, but different in many others. We've created this document to help you better understand how they compare.
+linktitle: AWS CDK
+menu:
+  intro:
+    parent: vs
+    weight: 3
+---
+
+<style>
+    main table {
+        font-size: 0.94em;
+    }
+
+    main table th,
+    main table td {
+        width: 33.3%;
+    }
+</style>
+
+Choosing the right infrastructure as code tool is critical, and we want you to have the best information possible to make the choice that best fits your needs. We've created this document to help you understand how Pulumi compares to AWS Cloud Development Kit (CDK).
+
+## What is AWS CDK?
+
+AWS Cloud Development Kit (AWS CDK) is an open-source software development framework to define your cloud application resources using familiar programming languages. CDK is a transpiler that translates CDK apps written in programming languages into AWS CloudFormation YAML/JSON, which you use with CloudFormation to deploy resources.
+
+## Pulumi vs. CloudFormation: Similarities {#similarities}
+
+CDK and Pulumi have some similarities in that you can build, deploy, and manage infrastructure as code on AWS using familiar languages and software tools.
+
+Both CDK and Pulumi use a declarative infrastructure as code model that’s built on the concept of desired state and oriented around managing groups of related cloud resources known as stacks. Because CDK relies on CloudFormation as the deployment engine, it inherits the declarative and desired state model used by CloudFormation. Desired state is expressed by declaring one or more cloud resources in code that’s evaluated by the deployment engine, which in turn decides whether to create, update or destroy the resources of a given stack by comparing its last known state with the new state expressed in the code.
+
+Finally, CDK and Pulumi have similar concepts for modularity or higher-level abstractions that encapsulate multiple resources. These are known as constructs in CDK and component resources in Pulumi.
+
+## Pulumi vs. CloudFormation: Key Differences {#differences}
+
+First, CDK only supports AWS, while Pulumi supports over 60 cloud and SaaS providers. Second, although both CDK and Pulumi support languages like TypeScript, Python, Go, and .NET, the tools are fundamentally different in how they interpret the languages and deploy resources. Unlike Pulumi, whose open-source engine understands these languages, CDK is a transpiler (a.k.a., source-to-source compiler) that translates programs into AWS CloudFormation YAML/JSON. The resulting markup file is then submitted to the closed-source AWS CloudFormation service, which provides the deployment engine for provisioning resources.
```

Comment:
Would be good to add the fact we're adding more providers every day (and growing)